src/Faker/Generator.php

```diff
@@ -273,31 +273,44 @@ public function seed($seed = null)
         }
     }
 
-    public function format($formatter, $arguments = [])
+    public function format($format, $arguments = [])
```

Comment:
Since we are compatible with PHP 8, this is a backwards-compatibility break.

Perhaps we should consider using a `@no-named-arguments` annotation?

For reference, see

- https://github.com/sebastianbergmann/phpunit/commit/754f932203fdc567775ae1b7869f1b3d1d70e379
- https://github.com/vimeo/psalm/blob/211553c53f09fa278946fda37b99c2f8ad1c1cbd/docs/running_psalm/issues/ParamNameMismatch.md#no-named-arguments

Reply:
Yes, I dont think the parameter name should be part of our BC promise. 

For the record I think that is a mistake from PHP core.